core/classes.py

```diff
@@ -1,6 +1,89 @@
+from logging import Logger
+
 import discord
 from discord.ext import commands
 
+
 class Cog_Extension(commands.Cog):
-       def __init__(self,bot):
-            self.bot = bot
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+
+class Hybirdcmd_Aliases():
+    def __init__(self, lang, *cmdName: str):
+
+        self.cmdName = list(cmdName)
+        self.aliases = {}
+        self.description = {}
+        self.brief = {}
+
+        for i in cmdName:
+
+            try:
+                aliases = []
+                for alias in lang[i+'.aliases']:
+                    alias = alias.lower()
+                    if alias != i.lower():
+                        aliases.append(alias)
+                self.aliases[i] = aliases
+
+            except:
+                self.aliases[i] = []
+
+            try:
+                self.description[i] = lang[i+'.description']
+                # baro's descripiton is always too long for somereason
+                if i == 'baro':
+                    self.description[i] = 'no_description'
+            except:
+                self.description[i] = 'no_description'
+
+            try:
+                self.brief[i] = lang[i+'.brief']
+            except:
+                self.brief[i] = 'no_brief'
+
+        self.count = len(self.cmdName)
+        self.point = 0
+
+    def next(self):
+        self.point = self.point+1
+        if self.point == self.count:
+            self.point = 0
```

Comment:
why don't you just use `__name__` to get function name and use it for indexing?

Reply:
most function name is different from command name